app/tests/integration/nimbus/kinto/client.py

```diff
@@ -30,7 +30,7 @@ def _has_pending_review(self):
         return self._fetch_collection_data()["status"] == KINTO_REVIEW_STATUS
 
     def approve(self):
-        for i in range(60):
+        for _ in range(60):
```

Comment:
really??

Reply:
`_` is usually a placeholder for a value you must unpack but then don't care to reference again.  It's more of a [pattern](https://stackoverflow.com/a/5893946) than an actual syntax, but actually it is now a formal syntax for the new [match statement](https://docs.python.org/3/reference/lexical_analysis.html#soft-keywords)